### PR TITLE
Module linting: refactor parsing of input channel names.

### PR DESCRIPTION
**Pull request by ewels**
No more split, regexes FTW!
Fixes nf-core/tools#1542

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by mirpedrol — APPROVED** [review]
Tested with `nf-core modules lint busco` and it doesn't fail 👍

File: nf_core/modules/lint/main_nf.py [hunk 1]
```diff
@@ -260,26 +262,41 @@ def check_process_section(self, lines):
...
+
+    If more than one elements in channel should work with both of:
+        tuple val(meta), path(reads)
+        tuple val(meta), path(reads, stageAs: "input*/*")
+
+    If using a tuple, channel names must be in (parentheses)
+    """
+    inputs = []
+    # Remove comments and trailing whitespace
+    line, *_ = line_raw.partition("//")
```
Comment (Midnighter) [inline nf_core/modules/lint/main_nf.py:277]: How about?

```suggestion
    line = line_raw.split("//")[0]
```

**Review by Midnighter — APPROVED** [review]
One suggestion, looks good otherwise.

**Comment by Midnighter** [comment]
Sheesh, things move fast 🙂

**Comment by ewels** [comment]
hah, sorry! Yeah that's nicer 👍🏻

**Pull request merged**